Bolso invaders/main.py

```diff
@@ -118,16 +116,14 @@ def isCollision(enemyX, enemyY, bulletX, bulletY):
             if event.key == pygame.K_RIGHT:
                 playerX_change = 1
             if event.key == pygame.K_SPACE:
-                if bullet_state is "ready":
-                    bulletSound = mixer.Sound("laser.wav")
+                if bullet_state == "ready":
```

Comment:
`is` é diferente de `==`

Abra o python e rode uma linha de cada vez:

```python
# assert vai reclamar se o valor lógico for False
assert True
assert False

a = "batata"
b = "batata"
assert a == b
assert a is b
# vai dar tudo True acima, mas então:
x = "ba"
y = "tata"
assert a == x + y
assert a is x + y
```
Para entender porque isso acontece, é preciso entender um pouco de Object-oriented programming OOP e como, no Python, todas as coisas **são** objetos (in the OOP sense).